public/app/features/alerting/unified/components/folder-bulk-actions/FolderBulkActionsButton.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+
+import { config, locationService } from '@grafana/runtime';
+import { Dropdown, Menu } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+import MoreButton from 'app/features/alerting/unified/components/MoreButton';
+
+import { alertingFolderActionsApi } from '../../api/alertingFolderActionsApi';
+import { FolderAction, useFolderAbility } from '../../hooks/useAbilities';
+import { useFolder } from '../../hooks/useFolder';
+import { createRelativeUrl } from '../../utils/url';
+
+import { DeleteModal } from './DeleteModal';
+import { FolderActionMenuItem } from './MenuItemPauseFolder';
+interface Props {
+  folderUID: string;
+}
+
+export const FolderBulkActionsButton = ({ folderUID }: Props) => {
+  const [pauseSupported, pauseAllowed] = useFolderAbility(FolderAction.Pause);
+  const canPause = pauseSupported && pauseAllowed && false; // lets disable pause for now
```

Comment:
we will disable pause/unpause actions for this PR, as we depend on the new endpoints to be finished.